msteams-platform/tabs/how-to/create-channel-group-tab.md

```diff
@@ -1024,6 +1024,43 @@ You've completed the tutorial to build a tab app with Blazor.
 
 ::: zone-end
 
+## Migrate your configurable tab to static tab
```

Comment:
Consider adding a note here (similar to the one found in `create-personal-tab.md`)

> [!NOTE]
> * Extend your static tabs to group chat and meetings is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
> * To extend your static tabs to group chat and meetings, use the app manifest v1.16 or later.